### PR TITLE
API endpoint to edit client versions

### PR DESCRIPTION
**Pull request by knrafto**
I'm taking a stab at on automating ok-client releases (https://github.com/Cal-CS-61A-Staff/ok-client/issues/264) and realized we'd need this

### CODE REVIEW

File: server/settings/test.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 if not os.path.exists(STORAGE_CONTAINER):
     os.makedirs(STORAGE_CONTAINER)
 
-CACHE_TYPE = 'simple'
+CACHE_TYPE = 'null'
```
Comment (Sumukh) [inline server/settings/test.py]: Just curious: What was failing because of this?

Comment (knrafto) [inline server/settings/test.py]: The version test does GET /API/v3/versions/ again to see if it actually changed, but the result is cached from the first call. I'm not sure if there's another way around that

Comment (knrafto) [inline server/settings/test.py]: The right thing to do is invalidate the cache. I've pushed a fix

**Comment by knrafto** [comment]
I've made some major changes, mind taking another look?

**Pull request merged**